cmd/trtl/main.go

```diff
@@ -307,6 +386,48 @@ func status(c *cli.Context) (err error) {
 	return nil
 }
 
+// dbGet prints values from the trtl database given a set of keys.
+func dbGet(c *cli.Context) (err error) {
+	b64decode := c.Bool("b64decode")
+	for _, keys := range c.Args().Slice() {
+		var key []byte
+		if key, err = wire.DecodeKey(keys, b64decode); err != nil {
+			return cli.Exit(fmt.Errorf("could not decode key: %s", err), 1)
+		}
+
+		var resp *pb.GetReply
+		req := &pb.GetRequest{
+			Key: key,
+			Options: &pb.Options{
+				ReturnMeta: c.Bool("meta"),
+			},
+		}
+		if resp, err = dbClient.Get(context.TODO(), req); err != nil {
+			return cli.Exit(err, 1)
+		}
+
+		// TODO: Make the output look nicer
```

Comment:
Any reason you don't simply want to output the JSON representation of the request? I hesitate to print things nicely, because it makes it harder to pipe things into other commands, for example, if I wanted to save the data to a file, I might `trtl db:get mykey | base64 -d > myfile.blob` -- if we do want to print things nicely, then I propose that we have a `--format` flag that will allow us to select `pprint, json, raw, base64` etc that determines how the data will be output.

Reply:
I agree, JSON is probably sufficient for the default case.